public/app/plugins/datasource/grafana-azure-monitor-datasource/components/MetricsQueryEditor/DimensionFields.tsx

```diff
@@ -46,11 +62,17 @@ const DimensionFields: React.FC<DimensionFieldsProps> = ({ query, dimensionOptio
               menuShouldPortal
               placeholder="Field"
               value={filter.dimension}
-              options={dimensionOptions}
+              options={validDimensionOptions}
               onChange={(v) => onFieldChange(index, 'dimension', v.value ?? '')}
               width={38}
             />
-            <InlineLabel aria-label="equals">==</InlineLabel>
+            <Select
+              menuShouldPortal
+              placeholder="Operation"
+              value={filter.operator}
+              options={dimensionOperators}
+              onChange={(v) => onFieldChange(index, 'operator', v.value ?? '')}
+            />
```

Comment:
If you want to, you can take a look at how operators are rendered in the CloudMonitoring filter component. There the input control is replaced by a button. I think it looks pretty good in the scenarios where predefined non text values such as operators are listed. I think it would be nice if we could do the same here.

![Screenshot from 2022-04-22 09-23-04](https://user-images.githubusercontent.com/2388950/164627712-25b89e76-6791-4f3e-91cd-56fb87d63ce6.png)

https://github.com/grafana/grafana/blob/47427-additional-dimension-operations/public/app/plugins/datasource/cloud-monitoring/components/LabelFilter.tsx#L126

Reply:
This makes sense for consistency, will take a look at implementing this 😊

Reply:
On the question of creating a custom operator value if Azure adds a new operator - I don't believe this would work as the display name of the operator doesn't directly match up with the actual value (e.g. `==` in the frontend is translated to `eq`). We could alternatively use the actual API representation however this may not be as appealing. It wouldn't require a change in the backend as we set the appropriate API value in the frontend here: 
![image](https://user-images.githubusercontent.com/15019026/164699132-240a3bb5-cff0-4db9-8109-f8524876a582.png)

Reply:
Maybe for optional values, it could be fine that the label name and value are the same? `{ label: 'ew', 'value: 'ew }` Thoughts?

Reply:
I think that'd be a happy medium as, if the user is specifying the operator, the correct value should be passed to the backend.